### PR TITLE
Update outdated docs

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Update outdated docs that refer to Less and atomic design code we no longer have.

## How to test this PR

1. PR checks should pass.
2. Running the docs locally should have updated docs.

### CODE REVIEW

File: docs/running-virtualenv.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ The following yarn tasks are available:
 
 ```
 yarn scripts             # Build the Javascript with esbuild
-yarn styles              # Build the Less with esbuild w/ its PostCSSPlugin
+yarn styles              # Build the SCSS with esbuild and its PostCSS plugin
```
Comment (wpears) [inline docs/running-virtualenv.md:128]: Also line 131 of this file

File: docs/running-virtualenv.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ The following yarn tasks are available:
...
 yarn copy                # Move static files to the output directory
 yarn build               # Run scripts, styles, and copy along with app-specific scripts
 yarn watch               # Run the build then watch JS and LESS changes
```
Comment (wpears) [inline docs/running-virtualenv.md:131]:
```suggestion
yarn watch               # Run the build then watch JS and SCSS changes
```

File: stylelint.config.js [hunk 3]
```diff
@@ -9,25 +9,21 @@ declaration-block-no-redundant-longhand-properties -
 declaration-empty-line-before -
   Turned off.
   TODO: Turn on this rule and work out what style we want.
-declaration-property-value-no-unknown -
-  Turned off for Less per documentation guidance.
 media-feature-range-notation -
-  Prefer prefixed values, since Less doesn't support ranges.
-media-query-no-invalid -
-  Turned off because of https://github.com/ssivanatarajan/stylelint-less/issues/6
+  Prefer prefixed values, traditionally used since Less doesn't support ranges.
```
Comment (wpears) [inline stylelint.config.js:13]: Possibly revisit this?

Comment (anselmbradford) [reply stylelint.config.js:13]: Yea revisiting several of those options would be a good idea. Some of them may generate a lot that needs updating if they are changed.

**Review by wpears — APPROVED** [review]

**Pull request merged**